sw/airborne/modules/mav_course_exercise/mav_exercise.c

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (C) 2021 Matteo Barbera <matteo.barbera97@gmail.com>
+ *
+ * This file is part of Paparazzi.
+ *
+ * Paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * Paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+#include "mav_exercise.h"
+#include "subsystems/abi.h"
+#include "firmwares/rotorcraft/navigation.h"
+#include "state.h"
+#include "autopilot_static.h"
+#include <stdio.h>
+
+#define NAV_C // needed to get the nav functions like Inside...
+#include "generated/flight_plan.h"
```

Comment:
Maybe these functions should be made public ? Including the flight_plan with `NAV_C` flag several times can be a problem when using variables.

Reply:
Sound like a good idea. But could this be a new issue/PR?

Reply:
yes, better not change this before a release